src/test/java/tests/search/checkout/CheckoutSubmissionTest.java

```diff
@@ -0,0 +1,41 @@
+package tests.search.checkout;
+
+
+import api.CheckoutApi;
+import api.SiteApi;
+import factories.CardFactory;
+import org.junit.Assert;
+import org.junit.Test;
+import pages.*;
+import types.Card;
+
+public class CheckoutSubmissionTest {
+    WebSite site = new WebSite();
+    BrowsePage browsePage;
+    OfferForm offerForm;
+    AvailabilityPage availabilityPage;
+    CheckoutPage checkoutPage;
+    CardFactory cardFactory = new CardFactory();
+    Card card = cardFactory.build();
+    SiteApi siteApi = new SiteApi();
+    CheckoutApi checkoutApi = new CheckoutApi();
```

Comment:
Looks like quite a few of these things aren't used? Also generally it's good practice to specify `private` if they're only intended for use inside the class, and `final` if they're not intended to be reassigned.